framework/source/class/qx/ui/window/IWindowManager.js

```diff
@@ -32,7 +32,9 @@ qx.Interface.define("qx.ui.window.IWindowManager",
      * @param desktop {IDesktop} The connected desktop
```

Comment:
@pcdummy please also document here that null is explicitly allowed as parameter like
```javascript
      * @param desktop {IDesktop|null} The connected desktop
```